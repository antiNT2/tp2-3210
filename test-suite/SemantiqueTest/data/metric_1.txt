num a;


a = 1 + 1;