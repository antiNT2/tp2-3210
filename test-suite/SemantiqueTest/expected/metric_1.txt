{VAR:1, WHILE:0, IF:0, ENUM_VALUES:0, OP:1}